Sources/Mantis/Helpers/TransformRecord.swift

```diff
@@ -0,0 +1,102 @@
+import UIKit
+
+public enum TransformType: Int {
+    case resetTransforms
+    case transform
+}
+
+public protocol TransformDelegate : AnyObject {
```

Comment:
In Mantis, we use code rule which doesn't include space in front of `:`. I noticed you are following different rules. We need to change that to keep the code in Mantis consistent.

```Swift
TransformDelegate: AnyObject
```

Reply:
fixed